percy/snapshot.py

```diff
@@ -81,3 +81,28 @@ def percy_snapshot(driver, name, **kwargs):
     except Exception as e:
         print(f'{LABEL} Could not take DOM snapshot "{name}"')
         print(f'{LABEL} {e}')
+
+# Take screenshot on driver
+def percy_automate_screenshot(driver, name, **kwargs):
```

Comment:
Can we move screenshot to `screenshot.py`? Having it in `snapshot.py` is weird

Reply:
We can't this will break because app-percy has `screenshot.py` as well. User having both the sdks will have issue. That was the reason this was decided that we will keep it here. @chinmay-browserstack  please elaborate here what was decided for this.

Reply:
As you can see https://github.com/percy/percy-selenium-python/pull/39/files#diff-f2fb6eb5e7b1c1b3ed12f3a0e3d1063f92791909868b1aac393ab909869fe55dR25-R26 for appium also we have screenshot.py and function name is also present. now what python does is override files which have same name (appium and selenium sdk have same package name). so we can't create screenshot.py in selenium else things will break for user who have both packages